### ISSUE TITLE
Intermittent pre-commit failures on github actions.

### ISSUE THREAD

**Issue by greschd** [aiidateam/aiida-core#3688]
The following pre-commit error seems to occur with some probability on github actions:

```
Messages
========

aiida/schedulers/plugins/slurm.py
  Line: 586
    pylint: invalid-name / Attribute name "requested_wallclock_time_seconds" doesn't conform to '(([a-z][a-z0-9_]{2,30})|(_[a-z0-9_]*))$' pattern (col 16)
```

The error is explicitly silenced with a ``# pylint: disable=invalid-name``, and I haven't seen it occur locally. Feels related to #3668, in that it could have the same root cause.

**Comment by sphuber** [comment]
yeah, this is a pain in the ass and I cannot reproduce it locally either. Seems to be a bug in pylint because it is flat out ignoring the disable rule. The variable used to be in camel case so I changed it because I thought that was the problem and of course then it worked, .... but only for a while. I will try updating the dependencies, because anyway `prospector==1.2.0` has been released which hopefully fixes some issues.
